config/mirror.yml

```diff
@@ -0,0 +1,45 @@
+---
+:branch_mirror_defaults: {}
+:branch_mirror_overrides: {}
+:productization_name: manageiq
+:remotes:
+  :red_hat_cloudforms: git@github.com:RedHatCloudForms
+  :upstream: git@github.com:ManageIQ
+:repos_to_mirror:
+  cfme-cloud_services: :red_hat_cloudforms
+  cfme-migration_analytics: :red_hat_cloudforms
```

Comment:
We should drop these, then leverage config gem merging of yaml to allow extensions via something like mirror.custom.yml

Reply:
Ok, I wasn't sure if anything depended on these two

Reply:
Done